data/starts.txt

```diff
@@ -95,7 +95,10 @@ start "hai space"
 	set "hai space start"
 	# need to set it here to not have it trigger alongside the invisible mission
 	set "First Contact: Hai: offered"
+	set "First Contact: Hai: declined"
```

Comment:
well adding the decline on top of the offered is required so it's the same as normal play and we dont break plugins or any other missions that dont use the proper one (declined)

then there's even missions in the main game that make this mistake, and we should not set a bad example for one, and should also follow conventions (11 times out of 13 or smth it was offered and then we have a few weird declined in there)

Reply:
I don't think the mistakes that plugins make should have any bearing on the vanilla game. Even if plugins have already been updated in the fortnight since Hai start was merged, the onus is on them to keep up with unstable development if they want to.